stampit.js

```diff
@@ -45,7 +45,7 @@ function extractFunctions(arg) {
 
 function addMethods(fixed, methods) {
   var args = [fixed.methods].concat(methods);
-  mixer.mixinFunctions.apply(null, args);
+  mixer.mixin.apply(null, args);
```

Comment:
Which example needs this change? I'd like to take a close look at it, because using the proto for non-methods is a strong code smell. Maybe the example can be improved, instead of changing stampit behavior...

The original version of Stampit did not filter out non-functions. We trusted that the name `.methods()` was good enough.